### DISCUSSION TITLE
fre:ac will not put a colon into a file or path name

### DISCUSSION THREAD

**Discussion by strange-antiquark** [enzo1982/freac#340] [Help & Support]
It would be helpful to be able to do things like the album title as <composer>: <albumartist>: <album> 
The colons are stored in the file header info, but are stripped out of the file/pathnames stored on disc.

fre:ac version 1.1.6 running on openSUSE Leap 15.4

**Comment by strange-antiquark** [comment]
> It would be helpful to be able to do things like the album title as : : 

Bleh -- it didn't like the angle brackets.
make that (composer): (albumartist): (album)

**Reply by Lee-Carre** [reply]
You should be able to escape metacharacters to achieve literal usage, by prefixing with a back-slash. Thus:

> \\<foo\\>:\\<bar\\>

**Comment by Lee-Carre** [comment]
Fre:AC being originally for MS Windows, and the colon being a special/forbidden character in directory/file names (since it's used as the separator between volume mount-point letter, and the absolute path (eg; `C:\Users\foobar`)); this is not surprising. Plus, what happens when you want to share your audio with a host running MS Windows?

Fre:AC is playing it safe, and complying with Postel's Law.

Though, yes, ideally it should be platform-conditional. But, lowest-common-demoninator compatibility is why (even though Unicode is >20 years old) sometimes ASCII is the only safe subset of characters to use; because (almost) everything groks them.

You could use semi-colons, instead. Else, try the likes of FFmpeg and some scripting-fu. Or, use more generic punctuation, such as the em-dash (U+2014). Even Microsoft implement Unicode (and non-badly), so it's cross-platform.
